### ISSUE TITLE
Error: Prohibited file name:

### ISSUE THREAD

**Issue by benoitlemay** [liximomo/vscode-sftp#267]
## Expected Behavior
Download and Upload file to server

## Actual Behavior
Can't download or upload file

### Extension Logs
  [error] Error: Prohibited file name: \web\admin\user\templates\PAR\includes.group\_share.html
    at makeError (C:\Users\Benoit\.vscode\extensions\liximomo.sftp-1.3.0\node_modules\ftp\lib\connection.js:1067:13)
    at Parser.<anonymous> (C:\Users\Benoit\.vscode\extensions\liximomo.sftp-1.3.0\node_modules\ftp\lib\connection.js:113:25)
    at emitTwo (events.js:106:13)
    at Parser.emit (events.js:194:7)
    at Parser._write (C:\Users\Benoit\.vscode\extensions\liximomo.sftp-1.3.0\node_modules\ftp\lib\parser.js:59:10)
    at doWrite (_stream_writable.js:329:12)
    at writeOrBuffer (_stream_writable.js:315:5)
    at Parser.Writable.write (_stream_writable.js:241:11)
    at Socket.ondata (C:\Users\Benoit\.vscode\extensions\liximomo.sftp-1.3.0\node_modules\ftp\lib\connection.js:273:20)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at TCP.onread (net.js:560:20) transfer \web\admin\user\templates\PAR\includes.group\_share.html
 

## Specifications

  - Version: latest (august 2nd)
  - Platform: Win7 64bits

**Comment by ragokin** [comment]
probably the same as #266, #265

**Comment by Jany-M** [comment]
Similar error here.

```
[error] Error: Prohibited file name: \path\page-home.php
	at makeError (C:\Users\Jany\.vscode\extensions\liximomo.sftp-1.3.0\node_modules\ftp\lib\connection.js:1067:13)
	at Parser.<anonymous> (C:\Users\Jany\.vscode\extensions\liximomo.sftp-1.3.0\node_modules\ftp\lib\connection.js:113:25)
	at emitTwo (events.js:106:13)
	at Parser.emit (events.js:194:7)
	at Parser._write (C:\Users\Jany\.vscode\extensions\liximomo.sftp-1.3.0\node_modules\ftp\lib\parser.js:59:10)
	at doWrite (_stream_writable.js:329:12)
	at writeOrBuffer (_stream_writable.js:315:5)
	at Parser.Writable.write (_stream_writable.js:241:11)
	at Socket.ondata (C:\Users\Jany\.vscode\extensions\liximomo.sftp-1.3.0\node_modules\ftp\lib\connection.js:273:20)
	at emitOne (events.js:96:13)
	at Socket.emit (events.js:191:7)
	at readableAddChunk (_stream_readable.js:178:18)
	at Socket.Readable.push (_stream_readable.js:136:10)
	at TCP.onread (net.js:560:20) transfer w:\somefolder\page-home.php
[info] transfer w:\somefolder\page-home.php
```

Win10 x64

**Comment by benoitlemay** [comment]
Is there a fix?  Going back to previous version of vscode and or extension?
Benoit Lemay
Griffmédia
Tel.: 418.998.6481

**Comment by benoitlemay** [comment]
Seems to be fix int 1.3.1!! THANKS!
